var/spack/repos/builtin/packages/steps/package.py

```diff
@@ -12,51 +14,127 @@ class Steps(CMakePackage):
     homepage = "https://groups.oist.jp/cnu/software"
     git = "https://github.com/CNS-OIST/STEPS.git"
 
-    version("3.3.0", submodules=True)
-    version("3.2.0", submodules=True)
+    maintainers("tristan0x")
+
     version("develop", branch="master", submodules=True)
+    version("4.1.1", submodules=True)
 
-    variant("native", default=True, description="Generate non-portable arch-specific code")
+    variant(
+        "codechecks",
+        default=False,
+        description="Perform additional code checks like code formatting or static analysis",
+    )
+    variant(
+        "native",
+        default=False if sys.platform == "darwin" else True,
+        description="Generate non-portable arch-specific code",
+    )
     variant("lapack", default=False, description="Use new BDSystem/Lapack code for E-Field solver")
-    variant("petsc", default=False, description="Use PETSc library for parallel E-Field solver")
+    variant("distmesh", default=True, description="Add solvers based on distributed mesh")
+    variant("petsc", default=True, description="Use PETSc library for parallel E-Field solver")
     variant("mpi", default=True, description="Use MPI for parallel solvers")
+    variant("coverage", default=False, description="Enable code coverage")
+    variant("bundle", default=False, description="Use bundled libraries")
+    variant("stochtests", default=True, description="Add stochastic tests to ctests")
+    variant(
+        "build_type",
+        default="RelWithDebInfo",
+        description="CMake build type",
+        values=("Debug", "Release", "RelWithDebInfo", "MinSizeRel", "RelWithDebInfoAndAssert"),
+    )
+    variant(
+        "caliper", default=False, description="Build in caliper support (Instrumentor Interface)"
+    )
+    variant(
+        "likwid", default=False, description="Build in likwid support (Instrumentor Interface)"
+    )
 
+    # Build with `ninja` instead of `make`
+    generator = "Ninja"
```

Comment:
Generator is now a directive, se #35552

Reply:
Done in fcc66e8ef2
Thank you for the suggestion